lib/common

```diff
@@ -208,3 +208,10 @@ get_allocated_nodes() {
 
   get_allocated_nodes_retval=$allocated_nodes
 }
+
+# each pacemaker cluster should have one founder node
+get_cluster_founders() {
+
+  local founders=$(knife search node "pacemaker_founder:true" -a name | grep ^name: | cut -d : -f 2)
```

Comment:
It's worth mentioning that this will return nodes that used to be founders of a cluster but where the cluster got removed. I don't think it matters much for now, but it might later on. Would it make sense to only consider nodes where the pacemaker proposal is active?